src/components/Carousel/components/SlideImage/SlideImage.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import Image from '../../../Image';
+import AspectRatio from '../../../AspectRatio';
+
+import { ASPECT_RATIO } from '../../constants';
+
+const backgroundStyles = ({ theme }) => css`
+  label: carousel__slideimage;
+  background: ${theme.colors.n100};
+`;
+// fix AspectRatio when upgrade to latest @emotion dependency
+const StyledAspectRatio = styled(AspectRatio)(backgroundStyles);
+
+const imageStyles = css`
+  label: carousel__image;
+  img {
+    object-fit: cover;
```

Comment:
`object-fit` is only supported in Edge 16+, so this would need to be polyfilled. This brings up an interesting question: what browsers do we want to support in v2?

Reply:
This can be communicated in the docs regarding `Carousel` or any other component with latest features like "please include polyfill if you need support for old browsers".

As for browser support, I would vote for last 2 versions.

Reply:
Does this mean then Edge and 11?

Reply:
I'd say only Edge. 

I mean, the only project that is required to support old browsers is the dashboard, because of the Android webviews. I don't think that one project should hold back Circuit UI. I like @voronianski's suggestion to mention the required polyfills, either on a component level or globally, the same way React does.

Reply:
I'd say only Edge. 

I mean, the only project that is required to support old browsers is the dashboard, because of the Android webviews. I don't think that one project should hold back Circuit UI. I like @voronianski's suggestion to mention the required polyfills, either on a component level or globally, the same way React does.